.travis.yml

```diff
@@ -14,6 +14,9 @@ env:
         - SYMFONY_DEPRECATIONS_HELPER="max[self]=0&max[direct]=0"
 
 jobs:
+    allow_failures:
+        - php: nightly
+
```

Comment:
The feature freeze has started today if we believe https://wiki.php.net/todo/php80, which means I should remove this, right?

Reply:
I guess so. But it would be reintroduced later when PHP 8.1 becomes a thing.

Reply:
Ok, I dropped it.